src/swapnaWJun20/CountVariations.java

```diff
@@ -0,0 +1,46 @@
+package swapnaWJun20;
+
+public class CountVariations {
+
+	void checkInput(String name)
+	{
+		int dCount = 0;
+		int letterCount = 0;
+		int uCount = 0;
+		int lCount = 0;
+		int specialCount = 0;
+
+		System.out.println("Input: " + name);
+
+		for(int index=0;index<name.length();index++)
+		{
+			char ch = name.charAt(index); 
+
+			if(Character.isDigit(ch)) 
+				dCount++;
+			else if (Character.isLetter(ch)) 
+				letterCount++;
+			else
+				specialCount++;
+
+			if(Character.isLowerCase(ch)) 
+				lCount++;
+			else if(Character.isUpperCase(ch))
```

Comment:
No two separate if else block is not required. You can achieve output without using letterCount varibale and else if (Character.isLetter(ch))  block. Think !!!

Reply:
Updated changes done!!